gammapy/maps/axes.py

```diff
@@ -291,7 +301,9 @@ def format_plot_xaxis(self, ax):
                                         quantity=PLOT_AXIS_LABEL.get(self.name, self.name.capitalize())
                                         , unit=self.unit )
         ax.set_xlabel(xlabel)
-        ax.set_xlim(self.bounds)
+        xmin, xmax = self.bounds
+        if not xmin == xmax:
+            ax.set_xlim(self.bounds)
```

Comment:
Note that with center defined axis bounds will stop exactly at the first and last bin centers. Maybe one should add a margin?

Reply:
But that can be another PR

Reply:
Would prefer a separate OR for this...